model/model.go

```diff
@@ -837,3 +841,25 @@ type Metric struct {
 	// Count holds the metric observation count for the bucket.
 	Counts []uint64 `json:"counts,omitempty"`
 }
+
+type Faas struct {
```

Comment:
```suggestion
// FAAS holds Function-as-a-Service properties.
type FAAS struct {
```

Can we please capitalise, for consistency with apm-server?